README.md

```
@@ -106,6 +106,137 @@ The vault aggregation logic starts automatically:
+  "project": "fomo"
+}
+
+
+
+4. Managing Future Whitelist Files
+
+For future projects, simply add the new JSON files to the /src/static directory and run the import script again. Since each project is identified by the filename (e.g., projectB.json), the import script will handle them appropriately.
+
+Example:
+
+	1.	Add projectC.json to /src/static.
```

Markdownlint

Column: 1
Hard tabs

(MD010, no-hard-tabs)